### ISSUE TITLE
RFW0112: [Tracking STT and TTS models and creating documentation]

### ISSUE THREAD

**Issue by tenzin3** [OpenPecha/Requests#362]
# RFW0112: [Tracking STT and TTS models and creating documentation]

## Summary
Tracking the model version and its hyperparameter and documenting of hyperparameter tuning and experiments
Documentation is for both Technical/Non Technical

## Key Concepts
hyperparameter:
A hyperparameter is a machine learning parameter that is chosen before a learning algorithm is trained. Hyperparameters are used to improve the learning of a model. They control how the model is training and determine the values of model parameters that a learning algorithm ends up learning

## Context
Provide a clear description of the work, its importance, and any background information necessary for someone new to understand and complete the work.

## Outputs
documentation of hyperparameter tuning and experiments
Documentation :

Standardised version naming (version of dataset)
- model link
- Data content distribution (eg Buddhist literature )
- Hyperparameter tuned
- Evaluation Metric and History
- Graphs

## Inputs
The training data and benchmark data set with its distribution report.
The model that is being trained

## Timeline
Specify the expected delivery date for the project.

## References
Having an standardized documentation for all AI models.
[RFW for MT documentation](https://github.com/OpenPecha/Requests/issues/340)

**Comment by kaldan007** [comment]
timeline missing
